src/web/views/Components/Slider/props-data.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable */
+export default [
+  [
+    "Props",
+    "Type",
+    "Required",
+    "Default value",
+    "Description",
+    "Example",
+  ],
+  [
+    "min",
+    "Number, String",
+    "",
+    "null",
+    "Use this props to define min range.",
+    "0",
+  ],
+  [
+    "max",
+    "Number, String",
+    "",
+    "null",
+    "Use this props to define max range.",
+    "100",
+  ],
+  [
+    "step",
+    "Number, String",
+    "",
+    "null",
+    "You can change the default step increment.",
+    "10",
```

Comment:
`step` default value is **1** not **10**.

Reply:
The number 10 here is for the "example" part